pandas/tests/groupby/test_api_consistency.py

```diff
@@ -82,13 +77,10 @@ def test_frame_consistency(request, groupby_func):
 def test_series_consistency(request, groupby_func):
     # GH#48028
     if groupby_func in ("first", "last"):
-        msg = "first and last are entirely different between Series and groupby"
-        request.node.add_marker(pytest.mark.xfail(reason=msg))
-    if groupby_func in ("cumcount", "corrwith"):
-        msg = "Series has no such method"
-        request.node.add_marker(pytest.mark.xfail(reason=msg))
+        # first and last are entirely different between Series and groupby
+        return
 
-    if groupby_func == "ngroup":
+    if groupby_func in ("cumcount", "corrwith", "ngroup"):
```

Comment:
ive had these corrwith xfails on my radar for a bit.  if we remove the xfail, should we also say "no" to #32293?

Reply:
I don't think so; replacing them with the attribute check means that this test will still start failing if they are ever added, but without the perf impact of using an xfail.